app/helpers/tooltips_helper.rb

```diff
@@ -81,11 +81,19 @@ def item_count_tooltip
   def metadata_completeness_tooltip
     "For each field, the percentage of all your metadata records that have a non-null value for that field.
     These fields are used most heavily in our search and discovery systems,
-    and therefore a higher percentage of completeness can lead to more use.
-    If a document has mediaAccess, that means that it has either a IIIF manifest
+    and therefore a higher percentage of completeness can lead to more use."
+  end
+
+  def media_access_tooltip
+    "If a document has mediaAccess, that means that it has either a IIIF manifest
```

Comment:
Since this is the display text, spell out as "Media Access"? Also, should we use "item" instead of "document" (this line and line 93), as that is the DPLA terminology (if I'm understanding correctly)?

Reply:
:+1: